pages/blog/types-of-apis.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Types of APIs"
+date: "2021-11-16"
+author: "Micah Bello"
+seoDescription: "Learn more about the different types of APIs that exist. Grouparoo makes it easy to connect your data warehouse with tools' APIs. ✓ Start a free trial now!"
+pullQuote: "Learn more about the different types of APIs that exist. Grouparoo makes it easy to connect your data warehouse with tools' APIs."
+image: "types-of-apis/hive.jpg"
+tags: [data]
+---
+
+Application Programming Interfaces or APIs are an integral part of modern software development and enable a wide variety of applications and workflows. Enterprises are becoming increasingly reliant on APIs to effectively connect with partners and customers.
+
+APIs come in an array of types and protocols that work great in different scenarios. In this article, we’ll examine the different types of APIs used in software development today. Let’s get started.
+
+<Image
+  alt="Image of a bee hive"
+  src="types-of-apis/hive.jpg"
+  centered
+  height={730}
+  width={918}
+/>
+
+## Types of APIs
+
+APIs are often categorized based on their scope or level of access. So how many APIs are there? The four main API types are public, internal, partner, and composite APIs.
+
+### Public APIs
+
+A public or open API is one that’s designed for use by the public. As a result, such APIs are easily accessible by any developer or business seeking to utilize the data or functionality they provide.
+
+Public APIs can either be free or may require registration and a fee per call depending on the provider. Some examples of [public APIs](https://github.com/public-apis/public-apis) include Archive.org’s search and book APIs, Square’s payment API, and cryptocurrency APIs such as Coinbase’s API.
```

Comment:
Not blocking -- we link to the external APIs we mention below... might do so here too.

Reply:
Cool, made a few updates.